topics/introduction/tutorials/galaxy-intro-peaks2genes/tutorial.md

```diff
@@ -121,38 +136,37 @@ we also need a list of genes in mice, which we can obtain from UCSC.
 
 > ### {% icon hands_on %} Hands-on: Data upload from UCSC
 >
-> 1. In the tool menu, navigate to `Get Data -> UCSC Main - table browser`
+> 1. Search for `UCSC Main` in the tool search bar (top left)
 >
 >     ![UCSC Main tool in tools section](../../images/101_01.png)
 >
+> 2. Click on `UCSC Main` {% icon tool %}  
+>
 >     You will be taken to the **UCSC table browser**, which looks something like this:
 >
 >     ![UCSC table browser interface](../../images/intro_02.png)
 >
 > 2. Set the following options:
->     - **clade** to `Mammal`
->     - **genome** to `Mouse`
->     - **assembly** to `July 2007 (NCBI37/mm9)`
->     - **group** to `Genes and Gene Predictions`
->     - **track** to `RefSeq Genes`
->     - **table** to `refGene`
->     - **region** to `genome`
->     - **output format** to `BED - browser extensible data`
->     - **Send output to** to `Galaxy` checked
+>     - *"clade"*: `Mammal`
+>     - *"genome"*: `Mouse`
+>     - *"assembly"*: `July 2007 (NCBI37/mm9)`
+>     - *"group"*: `Genes and Gene Predictions`
+>     - *"track"*: `RefSeq Genes`
+>     - *"table"*: `refGene`
+>     - *"region"*: `genome`
+>     - *"output format"*: `BED - browser extensible data`
+>     - *"Send output to"*: `Galaxy`
```

Comment:
This now seems to describe a select box, while there are 3 checkboxes (`Galaxy`, `GREAT` and `GenomeSpace`) and only `Galaxy` needs to be checked..